tests/tracking/integration_test_utils.py

```diff
@@ -16,7 +16,6 @@
 
 def _await_server_up_or_die(port, timeout=60):
     """Waits until the local flask server is listening on the given port."""
-    print("Awaiting server to be up on %s:%s" % (LOCALHOST, port))
```

Comment:
Let's replace print functions in this file with logging methods.

Reply:
Done! I put INFO level logs